sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -230,8 +230,8 @@ async function main() {
   console.log(`Created on: ${response.createdOn}`);
   console.log(`Last modified: ${response.lastModified}`);
 
-  if (response.models) {
-    for (const submodel of response.models) {
+  if (response.subModels) {
```

Comment:
@jeremymeng can you rename this to submodels, no capitalization? I know the class name is CustomFormSubModel, but in snake case, this would be sub_models